src/js/app.js

```diff
@@ -55,7 +55,17 @@ app.config(['$stateProvider', '$urlRouterProvider', '$locationProvider', '$httpP
       })
       .state('about', {
         url: '/about',
-        templateUrl: 'partials/about.html',
+        templateUrl: 'partials/under_construction.html',
+        requireLogin: false
+      })
+      .state('about.team', {
+        url: '/about/team',
```

Comment:
This is actually mounted at `/about/about/team`. In UI-Router, dot notation is used to designate a state as a child of a parent state. When a state is a child of another, it inherits the parents route path. So the correct value to have here is actually just `/team`.